packages/devtools_shared/lib/src/memory/event_sample.dart

```diff
@@ -80,10 +78,10 @@ class ExtensionEvent {
 
   factory ExtensionEvent.fromJson(Map<String, dynamic> json) =>
       ExtensionEvent.custom(
-        json['timestamp'] as int,
-        json['eventKind'] as String,
-        json['customEventName'] as String,
-        json['data'] as Map<String, Object>,
+        json['timestamp'] as int?,
```

Comment:
Should the constructors (default and named) have the timestamp, eventKind and data be required so the the ? is necessary?

Reply:
The `ExtensionEvent.empty` constructor initializes these values to null, so that is currently the only reason they are nullable